k8s/models/policy_v1_pod_disruption_budget.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# -*- coding: utf-8
+
+# Copyright 2017-2019 The FIAAS Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+
+import six
+
+from .common import ObjectMeta
+from ..base import Model
+from ..fields import Field, ListField
+
+
+class LabelSelectorRequirement(Model):
+    key = Field(six.text_type)
+    operator = Field(six.text_type)
+    values = ListField(six.text_type)
+
+
+class LabelSelector(Model):
```

Comment:
I see that there are three other `LabelSelector` model classes which seem to represent the same [API resource](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.24/#labelselector-v1-meta) already:
- https://github.com/fiaas/k8s/blob/master/k8s/models/deployment.py#L28
- https://github.com/fiaas/k8s/blob/master/k8s/models/job.py#L26
- https://github.com/fiaas/k8s/blob/master/k8s/models/pod_disruption_budget.py#L33

Perhaps one step towards improving this situation could be to move this LabelSelector class into a new `meta_v1` module (to follow the naming convention) and use that here. At some point it would be good to switch to the same class in every model that uses LabelSelector, but since that is a breaking change I don't think it is necessary to do in this changeset. What do you think?

Reply:
i think it's a good idea, though we already have a common package which currently houses metav1 type objects. So i think it would go best in there. If we really want to call it meta_v1 we should move everything over there